### DISCUSSION TITLE
How to add URL in page description?

### DISCUSSION THREAD

**Discussion by mohsaied** [alshedivat/al-folio#2155] [Q&A] [answered]
How can I add URL in page description? I used html before but now it seems to be broken after the latest rebase?

old way: 

`See my <a href="https://scholar.google.com/citations?user=q4wBpWAAAAAJ">Google Scholar profile</a>`

new way: ??

**Comment by george-gca** [comment]
Where exactly? If it is in a `.md` file you could simply write it `[like this](http://www.google.com)` that renders [like this](http://www.google.com).

**Comment by mohsaied** [comment, marked as answer]
The issue was this line in metadata.html

`<meta name="description" content="{%- if page.description -%}{{ page.description }}{%- else -%}{{ site.description }}{%- endif -%}" />`

basically any html that I wrote in the page.description was picked up by this and rendered on the page at the top.

**Comment by lvanveen** [comment]
Sorry to re-open this discussion, but the proposed solution does not work for me. I have the same issue as @mohsaied when using HTML tags, and when I instead use the format [link name](https:// ...) it is not rendered at all. The result is live at 
https://lvanveen.github.io/publications/
I re-typed the links to make sure there are no hidden characters and ran the deployment several times, but to no avail. 
Is it an option to edit metadata.html to re-enable the use of HTML tags? That doesn't sound like the proper solution, though.

**Reply by lvanveen** [reply]
For now, I changed the one line the @moshaied included above the the version I found in his repo:

Does this simply comment the one line out? It works now with HTML tags, but I suppose this may break something else. Please let me know if there is a better solution.

**Reply by george-gca** [reply]
Can you test this solution? Replace this line in [_includes/metadata.liquid](https://github.com/alshedivat/al-folio/blob/f4a6e184a9a23c2a2070b0ea545d390fab1e5c98/_includes/metadata.liquid#L42):

```liquid
<meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
```

by this one that [converts the markdown](https://jekyllrb.com/docs/liquid/filters/) description

```liquid
<meta name="description" content="{% if page.description %}{{ page.description | markdownify }}{% else %}{{ site.description | markdownify }}{% endif %}">
```

Then write your description in markdown style. Something like

```markdown
See my [Google Scholar profile](https://scholar.google.com/citations?user=q4wBpWAAAAAJ)
```

**Reply by lvanveen** [reply]
No luck. I made those changes, but the page looks exactly like it did originally - the links aren't rendered and some of the page description appears at the top. I suppose moshaied's solution was to comment out that line and, for now, I will revert to that.
